modules/network/subnets.tf

```diff
@@ -49,8 +49,8 @@ resource "oci_core_subnet" "pods" {
 resource "oci_core_subnet" "int_lb" {
   cidr_block                 = local.int_lb_subnet
   compartment_id             = var.compartment_id
-  display_name               = var.label_prefix == "none" ? "int_lb" : "${var.label_prefix}-int_lb"
-  dns_label                  = var.label_prefix == "none" ? "intlb" : "${var.label_prefix}ib"
+  display_name               = var.label_prefix == "none" ? "int-lb" : "${var.label_prefix}-int-lb"

```

Comment:
Should display_name be added to ignore_changes?

Reply:
nope, display names are updatable and don't have to be unique: https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/core_subnet#display_name